src/FluentMigrator.Extensions.Postgres/PostgresGenerationType.cs

```diff
@@ -0,0 +1,24 @@
+#region License
+// Copyright (c) 2007-2019, FluentMigrator Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+namespace FluentMigrator
+{
+    public enum PostgresGenerationType
```

Comment:
It's self-expanatory, but a comment would be nice

Reply:
Comment added, as requested.